dotCMS/src/main/java/com/dotcms/cache/VanityUrlCache.java

```diff
@@ -0,0 +1,62 @@
+package com.dotcms.cache;
+
+import com.dotcms.vanityurl.model.CachedVanityUrl;
+import com.dotcms.vanityurl.model.VanityUrl;
+import com.dotmarketing.business.Cachable;
+import java.util.Set;
+
+/**
+ * This cache is used to map the Vanity URLs path to the Vanity Url
+ * content.
+ *
+ * @author oswaldogallango
+ */
+public abstract class VanityUrlCache implements Cachable {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert this "VanityUrlCache" class to an interface [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1694)